<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>video4linuxConfig</class>
 <widget class="QWidget" name="video4linuxConfig">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="brightness_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrast_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast:</string>
   </property>
  </widget>
  <widget class="QLabel" name="saturation_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="white_bal_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>White Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sharpness_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sharpness:</string>
   </property>
  </widget>
  <widget class="QLabel" name="exposure_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure:</string>
   </property>
  </widget>
  <widget class="QLabel" name="focus_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Focus:</string>
   </property>
  </widget>
  <widget class="QSlider" name="bright_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="contrast_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="saturation_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="white_bal_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2000</number>
   </property>
   <property name="maximum">
    <number>6500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sharpness_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="exposure_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>2047</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="focus_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="camera_view">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="bright_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="contrast_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="saturation_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="white_bal_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>6500</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="sharpness_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="exposure_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>2047</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="focus_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>250</number>
   </property>
  </widget>
  <widget class="QPushButton" name="setToDefault">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Set to Default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Init_Capture_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Init Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="gain_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain:</string>
   </property>
  </widget>
  <widget class="QSlider" name="gain_slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="gain_lcd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-Color: #000</string>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
